Build/BUILDmom6

```diff
@@ -95,10 +93,10 @@ mkmf -m Makefile -a ${SHiELD_SRC} -p libmom6.a -t "${BUILD_ROOT}/${TEMPLATE}" \
      -c "-DINTERNAL_FILE_NML -g -DMAX_FIELDS_=100 -DNOT_SET_AFFINITY -D_USE_MOM6_DIAG -D_USE_GENERIC_TRACER -DUSE_PRECISION=2 -I${NCEP_DIR}/libFMS/${COMPILER}/${BIT}" \
      -I${SHiELD_SRC}/FMS/axis_utils/include -I${SHiELD_SRC}/FMS/diag_manager/include -I${SHiELD_SRC}/FMS/fms/include -I${SHiELD_SRC}/FMS/fms2_io/include \
      -I${SHiELD_SRC}/FMS/horiz_interp/include -I${SHiELD_SRC}/FMS/include -I${SHiELD_SRC}/FMS/mpp/include -I${SHiELD_SRC}/FMS/sat_vapor_pres/include \
-     -I${SHiELD_SRC}/FMS/string_utils/include -I${SHiELD_SRC}/FMS/test_fms/fms/include -I${SHiELD_SRC}/mom6/src/MOM6/pkg/CVMix-src/include \
-     -I${SHiELD_SRC}/mom6/src/MOM6/src/framework ${BUILD_ROOT}/Build/mom6/pathnames_mom6
+     -I${SHiELD_SRC}/FMS/string_utils/include -I${SHiELD_SRC}/FMS/test_fms/fms/include  \
+     -I${SHiELD_SRC}/MOM6/src/framework ${BUILD_ROOT}/Build/mom6/pathnames_mom6
 
-make -j8 REPRO=Y OPENMP=Y AVX=Y NETCDF=3 Makefile libmom6.a >& Build_mom6.out
+make -j8 REPRO=Y AVX=Y NETCDF=3 Makefile libmom6.a >& Build_mom6.out
```

Comment:
Does MOM6 not support OpenMP or is there just not a compiler flag for this?

Reply:
If used in debug mode, it was causing that noise in the ocean fields, which apparently was known issue.
In repro mode, parsing some files was causing a crash and it looks like compiling without openmp is solving it, although I believe it requires further investigation.

Reply:
Thank you for describing this issue. Is this an issue with the coupler or with MOM6?

Reply:
It is from a MOM6 file called MOM_file_parser

Reply:
MOM6 doesn't do OpenMP well at all, so this is a good choice.